file/types.go

```diff
@@ -523,17 +524,20 @@ type Info struct {
 	Defaults     KongDefaults `json:"defaults,omitempty" yaml:"defaults,omitempty"`
 }
 
+// Kong kong model
 // +k8s:deepcopy-gen=true
 type Kong struct {
 	Service *FService `json:"service,omitempty" yaml:"service,omitempty"`
 }
 
+// Implementation implementation model
 // +k8s:deepcopy-gen=true
 type Implementation struct {
 	Type string `json:"type,omitempty" yaml:"type,omitempty"`
 	Kong *Kong  `json:"kong,omitempty" yaml:"kong,omitempty"`
 }
 
+// FServiceVersion file service version model
```

Comment:
```suggestion
// FServiceVersion represents a Service version in Konnect. The type is duplicated because
// only a single document is exported in file while the API allows for multiple documents. 
```